astro_2.0/features/DaoDashboard/components/DashboardChart/components/ChartLegend/ChartLegend.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import { render } from 'jest/testUtils';
+
+import { ChartLegend } from 'astro_2.0/features/DaoDashboard/components/DashboardChart/components/ChartLegend/ChartLegend';
+
+describe('Comment', () => {
```

Comment:
I consider main description and test descriptions should be fixed.

Reply:
fixed